app/views/users/liked.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div class="col md-6 offset-3">
```

Comment:
great job formatting your user pages! Tiny error with "col md-6 offset-3". It should be "col-md-6 offset-3" (with the added dash in between col and md). Double check your show.html.erb for that same error as well!